draft-ietf-taps-interface.md

```diff
@@ -1904,47 +1904,72 @@ options.
+  multiplexing SHOULD assign the DSCP Default Forwarding {{?RFC2474}} PHB; when
+  the Connection is multiplexed, the guidelines in section 6 of {{?RFC7657}}
+  apply.
+
+  Scavenger/Bulk: : The application is not interactive. It expects to send
+  and/or receive data without any urgency. This can, for example, be used to
+  select protocol stacks with scavenger transmission control and/or to assign
+  the traffic to a lower-effort service. Transport system implementations that
+  map the requested capacity profile onto per-connection DSCP signaling without
+  multiplexing SHOULD assign the DSCP Less than Best Effort
+  {{?LE-PHB=I-D.ietf-tsvwg-le-phb}} PHB; when the Connection is multiplexed, the
+  guidelines in section 6 of {{?RFC7657}} apply.
+
+  Low Latency/Interactive: : The application is interactive, and prefers loss to
+  latency. Response time should be optimized at the expense of bandwidth
+  efficiency and delay variation when sending on this connection. This can be
+  used by the system to disable the coalescing of multiple small Messages into
+  larger packets (Nagle's algorithm); to prefer immediate acknowledgment from
+  the peer endpoint when supported by the underlying transport; and so on.
+  Transport system implementations that map the requested capacity profile onto
+  per-connection DSCP signaling without multiplexing SHOULD assign the DSCP
+  Expedited Forwarding {{?RFC3246}} PHB; when the Connection is multiplexed, the
+  guidelines in section 6 of {{?RFC7657}} apply.
+
+  Low Latency/Non-Interactive: : The application prefers loss to latency but is
+  not interactive. Response time should be optimized at the expense of bandwidth
+  efficiency and delay variation when sending on this connection.Transport
+  system implementations that map the requested capacity profile onto
+  per-connection DSCP signaling without multiplexing SHOULD assign a DSCP
+  Assured Forwarding (AF21,AF22,AF23,AF24) {{?RFC2597}} PHB; when the Connection
+  is multiplexed, the guidelines in section 6 of {{?RFC7657}} apply.
+
+  Constant-Rate Streaming: : The application expects to send/receive data at a
+  constant rate after Connection establishment. Delay and delay variation should
+  be minimized at the expense of bandwidth efficiency. This implies that the
+  Connection may fail if the desired rate cannot be maintained across the Path.
+  A transport may interpret this capacity profile as preferring a circuit
+  breaker {{?RFC8084}} to a rate-adaptive congestion controller. Transport
+  system implementations that map the requested capacity profile onto
+  per-connection DSCP signaling without multiplexing SHOULD assign a DSCP
+  Assured Forwarding (AF31,AF32,AF33,AF34) {{?RFC2597}} PHB; when the Connection
+  is multiplexed, the guidelines in section 6 of {{?RFC7657}} apply.
+
+  High Throughput Data: : \[EDITOR'S NOTE: Gorry wants this, but bht has no idea
+  how to implement it beyond DSCP AF1n {{?RFC2597}}. Cut, or specity how this is
```

Comment:
Sp: specity => specify

Reply:
Yeah, not really sure what to do with "High Throughput". Perhaps we can just specify that this enum list of profiles is allowed to grow in the future when we have better ideas?

Reply:
I'm concerned with us simply shrugging off stuff that has been previously defined in the IETF; the "High Throughput" service class is described in RFC 4594, Section 4.8. While I personally also find the service as described there a little awkward, I find it hard to believe that RFC 4594 was simply published with no prior discussion. If you want something newer, it's also discussed in RFC 8325. By mapping it to AF1n as described in RFC 4594, it seems simple enough to implement.

So, I'd rather keep it with a reference to this RFC and this simple mapping. But I'm not going to fight for it!

Reply:
If we want to keep it, we need text as to why an application would want this.